src/css/components/_slider.scss

```
@@ -0,0 +1,95 @@
+.sliderOverlay {
+    position          : fixed;
+    top               : 0;
+    left              : 0;
+    right             : 0;
+    bottom            : 0;
+    background-color   : rgba(0, 0, 0, 0.75);
+    width: $master-width;
+    height: $master-height;
+    margin: 25px;
+    z-index:1001;
+}
+
+.sliderModal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    right: auto;
+    bottom: auto;
+    @include transform(translate(-50%, -50%));
+    border: 1px solid #ccc;
+    overflow: auto;
+    -webkit-overflow-scrolling: 'touch';
+    border-radius: 4px;
+    outline: none;
+    padding: 20px;
+    min-width: 50%;
+
```

```suggestion
    max-width: 90%;
```
one more minor change, just so the modal doesn't go beyond the bounds of the HMI screen